Packages/com.unity.inputsystem/InputSystem/Editor/PropertyDrawers/InputActionAssetProperty.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using UnityEngine.Serialization;
+
+namespace UnityEngine.InputSystem.Editor
+{
+    [Serializable]
+    public struct InputActionAssetProperty
```

Comment:
Summary of problems I could see with this new type:
- This type is another type than InputActionAsset which implies we do not benefit from automatic property drawers on any existing code where InputActionAsset is a property, e.g. components in other packages or projects. Still I understand that you need a way to tell whether a certaain InputActionAsset is the global one or not, hence the bool.
- Given this solution what would happen if multiple InputActionAssetProperties exist which have the bool set to true but pointing to different assets? This type would allowing setting the bool from user code which may lead to  unexpected effects.

Reply:
Good points, thanks.
From my view, point 2 is the more critical from an API perspective.
We could probably leave with point 1 as we would only update our known components, leaving users to do what they wanted. Not desirable, but reasonable enough if there.
Based on your comments, I'll try again implementing this to `InputActionAsset` instead of creating a new type.